operator/internal/manifests/internal/config/loki-config.yaml

```diff
@@ -201,6 +201,8 @@ limits_config:
   cardinality_limit: {{ .Stack.Limits.Global.QueryLimits.CardinalityLimit }}
   max_streams_matchers_per_query: 1000
   query_timeout: {{ .Stack.Limits.Global.QueryLimits.QueryTimeout }}
+  volume_enabled: true
+  volume_max_series: 1000
```

Comment:
Didn't know that this is a new config option in the Loki `limits_config`. We need to expose this also in the LokiStack LimitsSpec here: https://github.com/grafana/loki/blob/main/operator/apis/loki/v1/lokistack_types.go#L607-L646. I suggest to call this field `MaxVolumeSeries`.